.taskcluster.yml

```diff
@@ -206,6 +210,35 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-pull-request" && "Train with task cluster:" in pr_description'
+        then:
+          dependencies:
+            - { $eval: as_slugid("docker_build") }
+          taskId: { $eval: as_slugid("train_on_taskcluster") }
+          created: { $fromNow: "" }
+          deadline: { $fromNow: "1 day" }
+          provisionerId: proj-bugbug
+          workerType: compute-smaller
+          payload:
+            maxRunTime: 10800
+            image: python:3.10
```

Comment:
We could instead rely on the image that we already built. In that image, we have the dependencies already installed and the repository is cloned.

Reply:
This is an example of using an image built from an earlier task: https://github.com/mozilla/task-boot/blob/bd37ee306ba4b40f0d21818e57b59b55579a63f3/.taskcluster.yml#L143.

Reply:
Oh yes! I think we have to rely on the image from `docker_build`